doc/content/getting_started/input_syntax.md

```diff
@@ -601,8 +608,9 @@ values during the simulation, which may be desired when running test runs.
 
 ## Debug Block id=debug
 
-This simply tells Moltres to print the variable residual norms during the
-non-linear solve.
+This parameter simply tells Moltres to print the variable residual norms during the
```

Comment:
```suggestion
This parameter tells Moltres to print the variable residual norms during the
```
Not needed, but feel free to ignore it feels like I'm erasing your voice from the docs.